.github/workflows/publish.yml

```diff
@@ -0,0 +1,17 @@
+name: Publish
+
+on:
+  workflow_dispatch:
+  workflow_run:
+    workflows: [Continuous integration]
```

Comment:
I wonder if we should rename the `Continuous integration` workflow to `CI`, so it is consistent with our other gems?